### ISSUE TITLE
Post Measures - Support for TLS 1.2

### ISSUE THREAD

**Issue by deepakrb** [librato/librato-java#48]
Since the drop of TLS 1.0 support I am unable to use the `postMeasures` endpoint without a Socket Exception. This is `librato-java` `2.1.2`

```
java.lang.RuntimeException: java.net.SocketException: Connection reset
at com.librato.metrics.client.DefaultPoster.post(DefaultPoster.java:63)
at com.librato.metrics.client.LibratoClient$4.call(LibratoClient.java:94)
at com.librato.metrics.client.LibratoClient$4.call(LibratoClient.java:87)
at java.util.concurrent.FutureTask.run(FutureTask.java:262)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
at java.lang.Thread.run(Thread.java:745)
```

Are there plans to support this or is there another way to post measures which doesn't have this error?

**Comment by mheffner** [comment]
@deepakrb What version of the JVM are you running?

**Comment by deepakrb** [comment]
It was 1.7, could that have something to do with it?

**Comment by mheffner** [comment]
@deepakrb Which version? Can you upgrade: https://stackoverflow.com/a/46582752

**Comment by deepakrb** [comment]
I'm going to close this since its no longer an issue (moved to the rest endpoints). I didn't manage to test the fix but I was running `Java 1.7.0_161` which should be within range of TLS1.2 support.
